config/initializers/govuk_publishing_components.rb

```diff
@@ -0,0 +1,12 @@
+if defined?(GovukPublishingComponents)
```

Comment:
Again, I find it unclear in which situations this shouldn't be defined.

Reply:
Oh, is it only defined in Development?

Reply:
Yes, we only include this Gem in Development and on Heroku